packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1758,10 +1873,16 @@ class MaxTileWidthGrid extends GridRenderObjectWidgetBase {
   }
 }
 
-/// Supplies per-child data to the grid's [GridDelegate].
-class GridPlacementData<DataType, WidgetType extends RenderObjectWidget> extends ParentDataWidget<WidgetType> {
-  GridPlacementData({ Key key, this.placementData, Widget child })
-    : super(key: key, child: child);
+/// A widget that controls the placement of a child in a grid.
+class GridPlacementData<DataType> extends ParentDataWidget<GridRenderObjectWidget> {
```

Comment:
If CustomGrid is going to point here for details, we should add a lot more detail here.